src/java.base/share/classes/sun/nio/fs/PollingWatchService.java

```diff
@@ -115,7 +115,7 @@ WatchKey register(final Path path,
             throw new IllegalArgumentException("No events to register");
 
         // Extended modifiers may be used to specify the sensitivity level
-        int sensitivity = 10;
+        int sensitivity = 1;
```

Comment:
Perhaps a (static final) constant such as `DEFAULT_SENSITIVITY` should be defined and used here and at line 311?

Reply:
Agreed :-)